README.md

```diff
@@ -133,24 +133,6 @@ See https://github.com/cozy/react-native-webview/blob/cozy_main/docs/Debugging.m
 - Then run ./scripts/build-debug-offline-apk.sh
 - The output will give you the path to the apk
 
-## How to enable Flagship certification
```

Comment:
I'm wondering if we should keep this section and talk about the google project json file instead? But I wouldn't know what to say as I don't know which part of the file is useful for the certification and which one is useful for push notifications

Reply:
- [ ] Will update